lua-bot-api.lua

```diff
@@ -964,6 +968,54 @@ end
 
 M.getChatMember = getChatMember
 
+local function deleteWebhook( )
```

Comment:
These functions are not necessary since `lua-telegram-bot` does not work with webhooks. It might create confusion and thus they should be removed.